.github/workflows/ci.yml

```diff
@@ -192,23 +191,18 @@ jobs:
       - name: Build
         uses: ./.github/actions/prepare-build
 
-      - name: Install Cypress
-        run: yarn cypress install
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
 
-      - env:
-          DEBUG: '@cypress/github-action'
-        name: Cypress run
-        uses: cypress-io/github-action@v2
-        with:
-          project: ./packages/website
-          start: yarn start
+      - name: Run Playwright tests
+        run: yarn playwright test
```

Comment:
my only issue with this is the output is pretty sparse:
<img width="718" alt="image" src="https://user-images.githubusercontent.com/7462525/195617446-462898b7-c9b7-4e8a-a4bf-32c9642470b7.png">

According to [this doc](https://playwright.dev/docs/next/test-reporters#list-reporter) this should include a bit more output for us

```suggestion
        run: yarn playwright test --reporter=list
```